### DISCUSSION TITLE
how to fix this problem

### DISCUSSION THREAD

**Discussion by REXMADE** [bnpr/Malt#369] [Q&A]
I'm new at this 
I downloaded the latest version
how can I fix it?
Sorry for my bad english 
thanks everyone
![image](https://user-images.githubusercontent.com/86163727/181825659-ee081512-4caa-444a-8a7a-2a7205a6cac7.png)

**Comment by pragma37** [comment]
See https://github.com/bnpr/Malt/issues/300#issuecomment-1126950272
